### DISCUSSION TITLE
Raycast with specific distance?

### DISCUSSION THREAD

**Discussion by LittleCodingFox** [jrouwe/JoltPhysics#578] [Q&A] [answered]
Hello,
I'm new to JoltPhysics and am trying to do a raycast with a specific distance, since it seems that currently NarrowPhaseQuery only handles up to a distance of 1.

Is there a way I can do this?

Thank you!

**Comment by DocAce** [comment, marked as answer]
I'm not sure where you're seeing such a limitation, but you can for sure do raycasts with any length. The direction is the vector from the raycast origin to the target position and does not need to be normalized.

**Reply by LittleCodingFox** [reply]
Oh I thought it needed to be normalized! My test works now, tyvm!
